configs/stage/roles/ocm.json

```diff
@@ -0,0 +1,52 @@
+{
+  "roles": [
+    {
+      "name": "OCM Cluster Editor",
+      "display_name": "Cluster Editor",
+      "description": "Grants permission to edit cluster",
+      "system": true,
+      "platform_default": false,
+      "version": 2,
+      "external": {
+        "id": "ClusterEditor",
+        "tenant": "ocm"
+      }
+    },
+    {
+      "name": "OCM Cluster Provisioner",
+      "display_name": "Cluster Provisioner",
+      "description": "Grants permission to provision clusters",
+      "system": true,
+      "platform_default": false,
+      "version": 2,
+      "external": {
+        "id": "ClusterProvisioner",
+        "tenant": "ocm"
+      }
+    },
+    {
+      "name": "OCM Cluster Viewer",
+      "display_name": "Cluster Viewer",
+      "description": "Grants permission to view clusters",
+      "system": true,
+      "platform_default": false,
+      "version": 2,
+      "external": {
+        "id": "ClusterViewer",
+        "tenant": "ocm"
+      }
+    },
+    {
+      "name": "OCM Organization Admin",
+      "display_name": "Organization Admin",
```

Comment:
Similar to above, it might be a bit confusing if OCM is left off the `display_name` and even `description` fields here, since these roles will be available in the ConsoleDot UI. It may mislead them to thinking they'll have org admin permissions over ConsoleDot resources?

Reply:
It wasn't clear to me from the user's perspective whether or not OCM is something they care about, or if it's just an implementation detail we want to obscure.  Thinking it over, it's probably better to go with including "OCM" for now; if it causes confusion, it will be more easily remedied than if it's absence caused confusion.